bucket/tikzit.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "2.1.6",
+    "description": "Graphical tool for rapidly creating graphs and string diagrams using PGF/TikZ",
+    "homepage": "https://tikzit.github.io/",
+    "license": "AGPL-3.0-or-later",
+    "architecture": {
+        "64bit": {
```

Comment:
Where is stated it is 64 bit??

![image](https://user-images.githubusercontent.com/13260377/123854156-9b23e800-d91e-11eb-8549-14661293c012.png)

Reply:
Fixed.